src/index.html

```diff
@@ -12,11 +12,50 @@
     />
     <title>Moyo header</title>
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
-      href="./style.css"
+    />
+    <link
+      rel="stylesheet"
+      href="style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo">
+        <a href="/">
+          <img
+            src="images/logo.png"
+            alt="Логотип компанії MOYO"
+          />
+        </a>
```

Comment:
Remember about correct indentation between parent and child elements. The `a` tag should be indented two spaces inside the `div` tag.